pandas/tests/io/test_parquet.py

```diff
@@ -483,13 +483,18 @@ def test_categorical(self, fp):
         check_round_trip(df, fp)
 
     def test_datetime_tz(self, fp):
-        # doesn't preserve tz
-        df = pd.DataFrame({'a': pd.date_range('20130101', periods=3,
-                                              tz='US/Eastern')})
+
+        # generic test data
+        df = pd.DataFrame({'dt': pd.date_range('20130101', periods=3)})
+
+        # fastparquet supports timezone since 0.1.4, and not before
+        import fastparquet
```

Comment:
you need to do this in a separate test. Have 1 more fp <= 0.1.3 and 1 for fp > 0.1.3 (essentially duplicate the test with 1 working for the old version and 1 for the new)

Reply:
need to add 1 build with fp pinned to 0.1.3, the so pin in requirements-2.7.sh

Reply:
@minggli we can indeed keep a test for the old behaviour (so only run for for older versions), but can you move the new behaviour to `test_basic` of this class (so we use the same pattern as in the pyarrow test), that is what I meant with my previous issue. 

> need to add 1 build with fp pinned to 0.1.3, the so pin in requirements-2.7.sh

@jreback We have one build for fastparquet 0.1.1 (that's the reason the tests were failing here in the original PR), so I would say that is enough?

Reply:
for sure.